crates/templates/src/manager.rs

```diff
@@ -181,6 +191,15 @@ impl TemplateManager {
         };
         let id = template.id();
 
+        if !install_template_list.is_empty() && !install_template_list.contains(&id.to_owned()) {
+            let message = format!("Skipping template {}...", id);
+            reporter.report(message);
+            return Ok(InstallationResult::Skipped(
+                id.to_owned(),
+                SkippedReason::IdNotIncluded,
+            ));
+        }
```

Comment:
Should this be moved into its function? Feels wrong to be passing in a vec of IDs to the `install_one` method.

Reply:
This is still installing one template though, so the name is no less correct.  But yeah, it's a bit weird that we call it for templates we don't want to install...!